mlflow/paddle/__init__.py

```diff
@@ -234,6 +240,20 @@ def load_model(model_uri):
             - ``models:/<model_name>/<model_version>``
             - ``models:/<model_name>/<stage>``
 
+    :param model: Required when loading a `paddle.Model` model. Must be an instance of
+                  `paddle.Model`. Note that the instance of `paddle.Model` must be set
+                  `training=True` when saved.
+    :param skip_mismatch: This only works when `model` is not None. Must be an instance
+                          of `paddle.Model`. Note that the instance of `paddle.Model`
+                          must be set `training=True` when saved. Default is False. If
+                          set true, `load_model` will ignore the parameters whose shape
+                          or name mismatch with those in model files that are saved
+                          previously.
+    :param reset_optimizer: This only works when `model` is not None. Must be an instance
+                            of `paddle.Model`. Note that the instance of `paddle.Model` must
+                            be set `training=True` when saved. Default is False. If set true,
+                            the `load_model` will ignore the optimizer provided.
```

Comment:
Can we replace these arguments with `kwargs`?

Reply:
For the docstring, do you think I need to list all the keywords for the ```kwargs``` that can be passed to ```paddle.jit.save``` and ```model.save```

Reply:
No, it's not. You can just indicate that `kwargs` will be passed to `paddle.jit.save` or `model.load`.

Reply:
I see. Thank you so much!